### ISSUE TITLE
register 1629

### ISSUE THREAD

**Issue by DoNNyes89** [maedball/Homeassistant_Siemens_versicharge#1]
Hi, thank you for your job! 
From some day the register 1629 ( pause On - Off) doesn't work. I can't reading or writing from it. The other registers work normally. 
Are you aware of the problem?

**Comment by achgut** [comment]
Yes, I am. My evcc implementation did not work any more since Friday when I got the update to FW2.128.

Since Firmware Update 2.128, the pause register 1629 is not valid any more.  Please refer to the Modbus Map
[versicharge_wallbox_modBus_map_en-US (5).pdf](https://github.com/maedball/Homeassistant_Siemens_versicharge/files/12270263/versicharge_wallbox_modBus_map_en-US.5.pdf)

I got the following feedback from the technical support:
Pausing of the charging is now done with the MaxChargingCurrentSum Register (1633). A value of 0 pauses the charging, restart with setting a valid Amp rating (e.g. 10Amps). 

This works for me now.
